src/components/SuggestionsProvider/SuggestionsList.js

```diff
@@ -0,0 +1,121 @@
+import React from 'react';
+import { FlatList } from 'react-native';
+import PropTypes from 'prop-types';
+import styled from '@stream-io/styled-components';
+import { CommandsItem, MentionsItem } from '../AutoCompleteInput';
+
+const Wrapper = styled.TouchableOpacity`
+  height: ${({ height }) => height};
+  position: absolute;
+  width: 100%;
+  z-index: 90;
+  ${({ theme }) => theme.messageInput.suggestions.wrapper.css}
+`;
+
+const Container = styled.View`
+  background-color: white;
+  border-top-left-radius: 10px;
+  border-top-right-radius: 10px;
+  bottom: 10px;
+  height: ${({ length, theme }) =>
+    Math.min(
+      length * theme.messageInput.suggestions.container.itemHeight,
+      theme.messageInput.suggestions.container.maxHeight,
+    )}px;
+  margin-left: ${({ marginLeft }) => marginLeft}px;
+  position: absolute;
+  shadow-color: #000;
+  shadow-offset: 0px -3px;
+  shadow-opacity: 0.05;
+  width: ${({ width }) => width}px;
+  z-index: 100;
+  ${({ theme }) => theme.messageInput.suggestions.container.css};
+`;
+
+const Separator = styled.View`
+  height: 0px;
+  ${({ theme }) => theme.messageInput.suggestions.separator.css};
+`;
+
+const SuggestionsItem = styled.TouchableOpacity`
+  height: ${({ theme }) => theme.messageInput.suggestions.container.itemHeight};
+  justify-content: center;
+  ${({ theme }) => theme.messageInput.suggestions.item.css};
+`;
+
+const Title = styled.Text`
+  height: ${({ theme }) => theme.messageInput.suggestions.container.itemHeight};
+  font-weight: bold;
+  padding: 10px;
+  ${({ theme }) => theme.messageInput.suggestions.title.css};
+`;
+
+const SuggestionsHeader = ({ title }) => <Title>{title}</Title>;
+
+const SuggestionsList = (props) => {
+  const {
+    active,
+    backdropHeight,
+    handleDismiss,
+    marginLeft,
+    suggestions: { data },
+    suggestionsTitle,
+    width,
+  } = props;
+
+  const renderItem = ({ item }) => {
+    const {
+      componentType: Component,
```

Comment:
`componentType` doesn't exist in the propType below so should we add it so we know we allow that in the props when we TS this component